textbook/12/1/uniform.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>I suggest the use of parentheses instead of commas in your reference like so:</p><p> </p><p>The uniform distribution has both discrete (that is, containing finite or countably infinite elements in its sample space as we depict in this chapter) and continuous (that is, containing infinite elements in its sample space) versions. For more information on the continuous uniform probability distribution see XYZ</p><p> </p><p>I think the use of parenthesis helps for readability and breakdown so they can better understand what the sentence is saying.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>